functions/_fzf_functions

```diff
@@ -1,7 +1,15 @@
 #! /usr/bin/env sh
 
+# default fzf functions - `$(brew --prefix)/opt/fzf/shell/key-bindings.zsh`
+
 # <<<< widgets >>>>
 
+# list ignored files
+# basically just `fzf-file-widget` with `--no-ignore`
+fzf-ignored-file-widget() {
+  FZF_CTRL_T_COMMAND="${FZF_DEFAULT_COMMAND} --no-ignore" fzf-file-widget
```

Comment:
Interesting.  I confess to not understanding why this line of code even works - I'd expect... a semi-colon before fzf-file-widget.  And, wouldn't this change the value of FZF_CTRL_T_COMMAND for future uses, or are we in some kind of "subscope" that only lives for the life of the single execution?

Reply:
Ok, reading up, so, sure, we're just setting the FZF_CTRL_T_COMMAND locally, which doesn't affect the global environment.  But... no need for a semi-colon?

Reply:
yeah these assignments are in scope only for the specific command. This [StackOverflow post](https://unix.stackexchange.com/questions/126938/why-is-setting-a-variable-before-a-command-legal-in-bash) does a good job explaining what's going on.